src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        if (userService.findByEmail(email).getEmail() == email
+                && userService.findByEmail(email).getPassword() == password) {
```

Comment:
We can save the user that we found to a variable to avoid calling `findByEmail()` twice
We should also ensure that it is not null before we call getter methods to avoid a null pointer exception